tasks/plugins.yml

```diff
@@ -13,3 +13,29 @@
     repository: "{{ item.repository | default('') }}"
     user_install: no
   with_items: "{{ sensu_server_gems }}"
+# XXX ansible `gem` module does not have an option to use `--bindir`. when gem
+# is one bundled in the official package, binaries are installed in correct
+# `bindir`. but when system's `gem` is used, they are installed in gem's
+# directory not in $PATH
+# https://github.com/ansible/ansible-modules-core/issues/2911
+# https://github.com/ansible/ansible/issues/25142
+- name: Get plug-ins' binaries
+  shell: "{{ sensu_server_gem_binary }} content {{ sensu_server_gems | selectattr('state', 'match', '^present$') | map(attribute = 'name') | list | join(' ') }} | grep -E '/bin/[^\\]+'"
+  changed_when: false
+  register: register_gem_content_grep_bin
```

Comment:
`register_gem_content_grep_bin` is not a good name because it does not represent what the variable is.
`register_sensu_server_gems_present` or something should be used.